src/OpenTelemetry.Api/README.md

```diff
@@ -358,12 +358,12 @@ to be associated with `Activity`. There is no `Status` class in .NET, and hence
 Example:
 
 ```csharp
-activity?.SetTag("otel.status_code", 2);
+activity?.SetTag("otel.status_code", "Error");
 activity?.SetTag("otel.status_description", "error status description");
 ```
 
-StatusCodes can be 0, 1, 2 which corresponds to `Unset`, `Ok` and `Error`
-respectively from [StatusCode.cs](./Trace/StatusCode.cs)
+[`StatusCode`](./Trace/StatusCode.cs) is an enum with options for `Unset`,
```

Comment:
StatusCodes can be the strings "Unset", "Ok", "Error" only, which corresponds respectively to `Unset`, `Ok`, `Error` from [StatusCode.cs](./Trace/StatusCode.cs)

Reply:
Yeah, your other comment makes sense to me now, I'll update this.